pkgs/development/tools/misc/clojure-lsp/disable-check-toolchain.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/graalvm/native-unix-compile.sh b/graalvm/native-unix-compile.sh
+index ec0e49f..e19a120 100755
+--- a/graalvm/native-unix-compile.sh
++++ b/graalvm/native-unix-compile.sh
+@@ -17,6 +17,7 @@ CLOJURE_LSP_XMX=${CLOJURE_LSP_XMX:-"-J-Xmx4g"}
+ 
+ args=("-jar" "$CLOJURE_LSP_JAR"
+       "-H:+ReportExceptionStackTraces"
++      "-H:-CheckToolchain"
```

Comment:
Hey @babariviere sorry for didn't notice that before, but why this flag is needed? I'm the clojure-lsp maintainer and maybe we could add that flag on clojure-lsp itself if that works for Windows/Mac/Linux, then we don't need this patch :) WDYT?

Reply:
I pushed to master adding this flag as it seems to not be a problem adding it for other OSs

Reply:
Basically, GraalVM is trying to check the version of the compiler before building. For whatever reason, it doesn't work on macOS, so it aborts compilation.

I didn't find a way to fix it on GraalVM, so it was easier to add this flag. (since it compiles correctly with it)

Reply:
LGTM, on next clojure-lsp bump, we can remove this patch so, thanks!

Reply:
Thank you too!